packages/xs-extra/xapi-xenopsd.master/opam

```diff
@@ -3,16 +3,16 @@ maintainer: "xen-api@lists.xen.org"
 authors: "xen-api@lists.xen.org"
 homepage: "https://github.com/xapi-project/xenopsd"
 build: [
-	["oasis" "setup"]
 	["./configure"]
 	[make]
 ]
 install: [
- 	["ocaml" "setup.ml" "-install"]
+ 	["./setup.bin" "-install"]
 ]
 remove: [
         ["oasis" "setup"]
```

Comment:
I think we should also remove this line. (And in general, we should make sure that the `opam` files here and in the source repo will be exactly the same.)